Telega/TelegramClient.cs

```diff
@@ -42,8 +42,7 @@ ISessionStore sessionStore
             Updates = new TelegramClientUpdates(_bellhop);
         }
 
-        public void Dispose()
-        {
+        public void Dispose() {
```

Comment:
to be same as others

Reply:
if so we'll need to use this style for all methods
i use this style only if i have a method formatted like

```
void F(
    int a,
    int b
) {
    ...
}
```

can it be configured in .editorconfig?